### ISSUE TITLE
Question, usecase for this over eslint a11y?

### ISSUE THREAD

**Issue by noahehall** [reactjs/react-a11y#127]
in udacity's senior dev course the third project is all about accessibility.

Im wondering if this, or the eslint plugin is better to use, or will they both report the same info to me?

thx

**Comment by homekit-worker** [comment]
What is the difference between this project and eslint a11y?
to top

**Comment by erin-doyle** [comment]
Since no one has answered this yet.  This is what it says on the [eslint-plugin-jsx-a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y) repo:

> Note: This project does not replace react-a11y, but can and should be used in conjunction with it. Static analysis tools cannot determine values of variables that are being placed in props before runtime, so linting will not fail if that value is undefined and/or does not pass the lint rule.

So one does not replace the other but they are probably used best together.

**Comment by noahehall** [comment]
Thanks @erin-doyle !
